packages/ui/useScroll/useScroll.module.less

```diff
@@ -37,8 +37,4 @@
 		overflow: hidden;
 		flex: 1 1 100%;
 	}
-
-	.scrollContentWrapperNative {
```

Comment:
Let's remove this from the code too.
https://github.com/enactjs/enact/blob/master/packages/ui/useScroll/useScroll.js#L1521

Reply:
OK